pal/src/include/pal/virtual.h

```diff
@@ -105,108 +105,6 @@ BOOL VIRTUALOwnedRegion( IN UINT_PTR address );
 
 #ifdef __cplusplus
 }
-
-/*++
-Class:
-    ExecutableMemoryAllocator
-
-    This class implements a virtual memory allocator for JIT'ed code.
-    The purpose of this allocator is to opportunistically reserve a chunk of virtual memory
-    that is located near the coreclr library (within 2GB range) that can be later used by
```

Comment:
On XPLAT, both has the same backing proxy hence no difference apart from making our bootstrap process heavy and complicated.